toil_orchestrator/views/job_view.py

```diff
@@ -24,7 +24,6 @@ def validate_and_save(self, data):
         serializer = JobSerializer(data=data)
         if serializer.is_valid():
             response = serializer.save()
-            submit_jobs_to_lsf.delay(str(response.id))
```

Comment:
Just curious what this call would do previously. Does this mean when a job is updated in the admin it is re-submitted / re-run?

Reply:
It would submit jobs after its submitted. I think this should be submit_pending_jobs instead.

Reply:
Nvm, I see this is handled by: https://github.com/mskcc/ridgeback/pull/151/files#diff-f828b61ce92a6fae16a7925b2fe809e9c72f91114cb5ac45df05b1530816bb7eR89.